server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/tasks/on_cancel_representer.rb

```diff
@@ -27,7 +27,7 @@ def from_hash(hash, options={})
 
         def to_hash(*options)
           return nil if on_cancel_config.getTask.getTaskType.eql?("killallchildprocess")
-          TaskRepresenter.new(on_cancel_config.getTask).to_hash(url_builder: UrlBuilder.new)
+          TaskRepresenter.new(on_cancel_config.getTask).to_hash(url_builder: nil)
```

Comment:
`UrlBuilder` is  a test-only class. For production usage we need to pass along the controller. This was possibly added in error,  hence removing it. Also, a value of `nil` is sufficient as there are no embedded urls in TaskRepresenter or anything under it.